makefiles/doc-cmds.mk

```diff
@@ -0,0 +1,48 @@
+ifneq ($(PROCURSUS),1)
+$(error Use the main Makefile)
+endif
+
+SUBPROJECTS      += doc-cmds
+DOC-CMDS_VERSION := 55
+DEB_DOC-CMDS_V   ?= $(DOC-CMDS_VERSION)
+
+doc-cmds-setup: setup
+	$(call GITHUB_ARCHIVE,apple-oss-distributions,doc_cmds,$(DOC-CMDS_VERSION),doc_cmds-$(DOC-CMDS_VERSION))
+	$(call EXTRACT_TAR,doc_cmds-$(DOC-CMDS_VERSION).tar.gz,doc_cmds-doc_cmds-$(DOC-CMDS_VERSION),doc-cmds)
+	mkdir -p $(BUILD_STAGE)/doc-cmds/$(MEMO_PREFIX)$(MEMO_SUB_PREFIX)/{bin,libexec,share/man/man8}
+
+ifneq ($(wildcard $(BUILD_WORK)/doc-cmds/.build_complete),)
+doc-cmds:
+	@echo "Using previously built doc-cmds."
+else
+doc-cmds: doc-cmds-setup
+	mkdir -p $(BUILD_STAGE)/doc-cmds/$(MEMO_PREFIX)$(MEMO_SUB_PREFIX)/{bin,share/man/man1/}
+	cd $(BUILD_WORK)/doc-cmds; \
+	for bin in checknr colcrt; do \
+		$(CC) $(CFLAGS) $(LDFLAGS) -o $(BUILD_STAGE)/doc-cmds/$(MEMO_PREFIX)$(MEMO_SUB_PREFIX)/bin/$$bin $$bin/$$bin.c; \
+		$(INSTALL) -Dm644 $$bin/$$bin.1 $(BUILD_STAGE)/doc-cmds/$(MEMO_PREFIX)$(MEMO_SUB_PREFIX)/share/man/man1/; \
+	done; \
+	$(CC) $(CFLAGS) $(LDFLAGS) -lz makewhatis/makewhatis.c -o $(BUILD_STAGE)/doc-cmds/$(MEMO_PREFIX)$(MEMO_SUB_PREFIX)/libexec/makewhatis; \
+	$(INSTALL) -m755 makewhatis/makewhatis.local.sh $(BUILD_STAGE)/doc-cmds/$(MEMO_PREFIX)$(MEMO_SUB_PREFIX)/libexec/makewhatis.local; \
+	$(INSTALL) -Dm644 makewhatis/makewhatis.{,local.}8 $(BUILD_STAGE)/doc-cmds/$(MEMO_PREFIX)$(MEMO_SUB_PREFIX)/share/man/man8/; \
```

Comment:
Don't install `makewhatis(8)`, this version is only used by man 1.3, which we don't use. `man-db` does not use `makewhatis`, and `mandoc` has it's own.